tutor/src/components/annotations/summary-page.jsx

```diff
@@ -39,6 +43,16 @@ export default class AnnotationSummaryPage extends React.Component {
     }
   }
 
+  prepareFocus() {
+    const {containerRef} = this;
+    const focusAnchor = document.createElement("a");
+
+    focusAnchor.setAttribute("href", "#");
+    containerRef.insertBefore(focusAnchor, containerRef.firstChild);
+    focusAnchor.focus();
+    focusAnchor.addEventListener("blur", () => containerRef.removeChild(focusAnchor), false);
+  }
```

Comment:
I'm afraid I don't see why we're why we're manually added the `focusAnchor` node?  Maybe there was a requirement that I missed somewhere?